jni/tools/native_loader/src/main/java/com/github/fmeum/rules_jni/RulesJni.java

```diff
@@ -228,7 +241,7 @@ private static void atExit() {
     if (!skipCleanup) {
       try (Stream<Path> tempFiles = Files.walk(tempDir)) {
         tempFiles.sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
-      } catch (IOException e) {
+      } catch (Exception e) {
```

Comment:
Just curious, what new kind of exception is this catching now?

Reply:
I wish I remembered exactly what it was called... but it was a file not found type exception that would happen sometimes when deleting the rules_jni/<some_file> because the whole rules_jni directory was already deleted in the for loop. It wasn't getting caught by IOException on Android, and I figured since clean up is best effort anyways it makes sense to expand this to catch every exception and eat it.